server/src/index.js

```diff
@@ -14,4 +15,9 @@ const server = new GraphQLServer({
   context: req => ({ ...req, db })
 });
 
-server.start(() => console.log('Server is running on http://localhost:4000'));
+server.express.use(cookieParser());
+
+server.start(
+  { cors: { origin: /^https?:\/\/(localhost:)(.*)/, credentials: true }},
```

Comment:
passing CORS options to `graphql-yoga`, any port on `localhost` so that both app and playground work